packages/test-cases/cases/apis/wasm-rs/asyncify/mutation/src/lib.rs

```diff
@@ -0,0 +1,127 @@
+pub mod w3;
+use polywrap_wasm_rs::JSON;
+pub use w3::mutation;
+pub use w3::*;
+pub use w3::imported::ethereum_mutation;
+pub use query::imported::ethereum_query;
+
+pub fn set_data_with_large_args(input: mutation::InputSetDataWithLargeArgs) -> String {
```

Comment:
If we wanted to keep this similar to the AssemblyScript version it'd be something like "Input_set_data_with_large_args" or maybe better "Input_SetDataWithLargeArgs"

Reply:
In idiomatic Rust, we'd have to suppress some compiler warnings (specifically, `#[warn(non_camel_case_types)]`) in order to have a type like `Input_SetDataWithLargeArgs`. That would also mean all Rust Polywrap devs must write unidiomatic Rust code while building a wrapper. Thus, I think we should keep to the convention.

Reply:
Oh damn, that's a shame. I do agree, InputSetDataWithLargeArgs is fine then